### PR TITLE
realtek: add support for I-O DATA BSH-G24MB

### PR DESCRIPTION
**Pull request by musashino205**
- u-boot-envtools: add support for I-O DATA BSH-G24MB
- realtek: add support for I-O DATA BSH-G24MB

### CODE REVIEW

File: target/linux/realtek/dts-5.10/rtl8382_iodata_bsh-g24mb.dts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+		reset {
+			label = "reset";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gpio1 {
```
Comment (hauke) [inline target/linux/realtek/dts-5.10/rtl8382_iodata_bsh-g24mb.dts:53]: This does not compile for me, I am getting this error:
``
dir/target-mips_4kec_musl/linux-realtek_rtl838x/image-rtl8382_iodata_bsh-g24mb.dtb /home/hauke/openwrt/openwrt/build_dir/target-mips_4kec_musl/linux-realtek_rtl838x/image-rtl8382_iodata_bsh-g24mb.dtb.tmp
Error: ../dts-5.10/rtl8382_iodata_bsh-g24mb.dts:53.1-7 Label or path gpio1 not found
FATAL ERROR: Syntax error parsing input tree
``

Comment (hauke) [reply target/linux/realtek/dts-5.10/rtl8382_iodata_bsh-g24mb.dts:53]: You have to do the same changes as done here: 4b31717fb031e6b99f476e79b5e8b3d6a8b0b9ff

Comment (musashino205) [reply target/linux/realtek/dts-5.10/rtl8382_iodata_bsh-g24mb.dts:53]: Oops, I forgot to update for it. I'll do...

**Comment by musashino205** [comment]
UPDATE:

- rebased from master
- add rtl8231 node to the device dts

**Comment by hauke** [comment]
Thank you for your patches, I applied them to master in 7b19770525bcb10c82bd6fd769b8db574ac9087f and 98113220fa87019c01dfcf08f41730f97323858a.

**Pull request closed without merging**